Draft-Accepted/RFCNNNN-Install-PowerShell.md

```diff
@@ -0,0 +1,176 @@
+---
+RFC: RFC00XX
+Author: Jason Helmick
+Status: Draft
+SupercededBy: N/A
+Version: 1.0
+Area: Core
+Comments Due: 6/30/2022
+Plan to implement: Yes
+---
+
+# Install-PowerShell7 Native Command
+
+This RFC proposes adding a `install-powershell7` native command to Windows to perform a simplified
+installation of the latest Long Term Service (LTS) release of PowerShell 7.
+
+## Motivation
+
+Many IT customers of PowerShell face a problem in which they don't control the landscape of software
+pre-installed on the Windows Servers they manage. Explicitly installing PowerShell is a blocker for
+some customers due to discoverability and some that have policies that don't allow for installing
+software onto Windows after initial provisioning.
+
+To compensate for these situations, customers will implement and use the "tools at hand" to perform
+management tasks. This leaves **Windows PowerShell 5.1**, not **PowerShell 7**, as the only
+automation solution available to many of them. While customers are missing out on useful new
+features that they may wish to have, they are certainly not receiving the performance and security
+improvements that come with each new release.
+
+This specification is an effort to help IT admins that wish to use new and updated releases of
+PowerShell 7. While this may not help customers that have post-deployment policy restrictions, it
+will help the majority of customers discover and update to the latest version of PowerShell for
+Windows.
+
+```
+As an admin,
+I can use Windows PowerShell 5.1 to provision multiple hosts with PowerShell 7 over PSRemoting,
+so that I can leverage new capabilities in PS7.
```

Comment:
200MB * 100 computers = 20GB will be downloaded directly from Internet. I'd prefer local cache.

Reply:
@iSazonov - Thank you for the comment. Deployments at scale are dependent on the customer infrastructure/apps and not a goal of this command.